library/network/src/modules/NetworkInterfaces.rb

```diff
@@ -263,11 +264,11 @@ def GetIbTypeFromSysfs(dev)
       sys_dir_path = Builtins.sformat("/sys/class/net/%1/", dev)
 
       if FileUtils.Exists(Ops.add(sys_dir_path, "bonding"))
```

Comment:
Why not to continue with conversion in the same way as above? I mean turn it into "#{sys_dir_path}/bondin" as you did above.